api/src/application/user/user-service.js

```diff
@@ -101,7 +101,7 @@ module.exports = (bookshelf, emailService, renderTemplate) => {
     },
 
     findByEmailForOidc: function(email) {
-      return bookshelf.model('user').forge({ email }).fetch()
+      return bookshelf.model('user').where('email', email).fetch()
```

Comment:
Ok that seems fine to me. Not sure why it doesn't break for others, but this change isn't a bad one.

I'll usually ask when I see changes that don't seem related to the issue being worked on.